pyaedt/edb_core/stackup.py

```diff
@@ -572,16 +576,31 @@ def import_stackup(self, fpath):
         return True
 
     @pyaedt_function_handler
-    def export_stackup(self, fpath, file_format="csv"):
-        """Export stackup definition to csv file.
+    def export_stackup(self, fpath, file_format="csv", include_material_with_layer=False):
+        """Export stackup definition to csv or json file.
 
         Parameters
         ----------
         fpath : str
-            File path to csv file.
+            File path to csv or json file.
         file_format : str, optional
             The format of the file to be exported. The default is ``"csv"``. Options are ``"csv"``, ``"xlsx"``.
+            ``"json"``.
+        include_material_with_layer : bool, optional.
+            Only used when json file are exported. ``True`` will include material definition inside layer ones.
+            ``False`` will keep the material definition section in the json file. Default value is ``False``.
```

Comment:
```suggestion
            Whether to include the material definition inside layer ones. This parameter is only used
            when a JSON file is exported. The default is ``False``, which keeps the material definition
            section in the JSON file. If ``True``, the material definition is included inside the layer ones.
```
"inside the layer ones" is confusing. What is "one"? Could it be changed to inside the layer definitions?